<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatDlg</class>
 <widget class="QDialog" name="ChatDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>对话</string>
  </property>
  <widget class="QGroupBox" name="gb_user">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>用户信息</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_details">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_input">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_send">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red</string>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_close">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pb_close</sender>
   <signal>clicked()</signal>
   <receiver>ChatDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
